ulib/LowStar.Monotonic.Buffer.fsti

```diff
@@ -960,6 +989,12 @@ val modifies_buffer_elim (#a:Type0) (#rrel #rel:srel a)
   (b:mbuffer a rrel rel) (p:loc) (h h':HS.mem)
   :Lemma (requires (loc_disjoint (loc_buffer b) p /\ live h b /\ modifies p h h'))
          (ensures  (live h' b /\ (as_seq h b == as_seq h' b)))
+         [SMTPatOr [
+             [ SMTPat (modifies p h h'); SMTPat (as_seq h b) ] ;
+             [ SMTPat (modifies p h h'); SMTPat (live h b) ];
```

Comment:
goal-directed?

Reply:
Re. the comments about these patterns: they are just copied from their corresponding `modifies_inert` lemmas, since `modifes_inert` is no longer there. I can try to see if changing them works.